camayoc/tests/qpc/yupana/utils.py

```diff
@@ -79,3 +75,109 @@ def post_file(
         verify=False,
     )
     return response
+
+
+def oc_setup(yupana_config):
```

Comment:
Where or when do you select the Openshift project for the tests in Yupana, after the login?

Reply:
Hmm... I don't think I do. Since I was just pulling from container logs so far, and I would explicitly state the application there, I don't think I've hit an issue with it (yet).

I'll add a line to select the project from the config here.

Reply:
I ended up switching this to be a fixture.